### DISCUSSION TITLE
ALPB Octanol (wet)

### DISCUSSION THREAD

**Discussion by bfmilne** [grimme-lab/xtb#556] [General]
Hi,

The documentation lists 'Octanol (wet)' as an option for the ALPB solvent model. How do you request this on the command line?

Cheers,
Bruce

**Comment by TyBalduf** [comment]
Probably a bit late to be of much help, but I believe the keyword for wet Octanol is `--alpb woctanol`. See [help msg](https://github.com/grimme-lab/xtb/blob/000b4f9998b10429b39202346e188b8ace3ce64f/src/xhelp.f90#L149)

**Comment by okbckim** [comment]
Hi

What is the difference between octanol and woctanol?

Thanks
-B
